README.md

```diff
@@ -76,7 +76,7 @@ Then, you can serialize the `Message` into `Vec<u8>` in encrypted form.
     let serialized_encrypted_message: Vec<u8> = encrypted_message.serialize()?;
 ```
 
-After your peer gets the binary, he or she can decrypt and deserialize it to `Message`.
+After your peer gets the binary, they can decrypt and deserialize it to `Message`.
```

Comment:
Thanks.

I'm not a native English speaker but I think we cannot use `they` here because it points to `your peer` (singular).
How about also changing your peer?

Reply:
`they` would be more accurate even for singular, but if you don't agree with the following text we can change to `your peer`. what do you think?

https://www.grammarly.com/blog/use-the-singular-they/?gclid=Cj0KCQjw_dWGBhDAARIsAMcYuJwBrHmnMZTZPID_nqCW3MBRKvKYb2HkfkLRppoWiaIXGUiVbnIw2pwaAvZ7EALw_wcB&gclsrc=aw.ds

Reply:
Oh, I have not known the usage of "they".
I accept your change. Thank you.

Reply:
Thanks!